pyscript.core/tests/integration/python/helper.js

```diff
@@ -0,0 +1,22 @@
+const qs = new URLSearchParams(location.search);
+
+// src= to NOT have a config
+const src = qs.has('src') ? qs.get('src') : './main.py';
+
+// config= to NOT have a src
+const config = qs.has('config') ? qs.get('config') : './settings.json';
+
+// terminal=0 to NOT have a terminal
+const terminal = qs.get('terminal') !== '0';
+
+// worker=1 to have a worker
```

Comment:
I find these two a bit confusing ... we could as well just use `has('terminal')` or `has('worker')` ... not sure why we need `=0` on terminal and `=1` on worker ... this is a nit though, still maybe you agree.

Reply:
This has more to do with my shonky JS "skillz". :wink: I'll fix this up... I agree with your proposed change. It's much more readable. :+1:

Reply:
it was probably me suggesting these two but that's because I forgot that `?terminal&worker` are perfectly valid too as query strings parameters